userspace/libsinsp/plugin.h

```diff
@@ -28,6 +28,21 @@ limitations under the License.
 
 #include "filter_check_list.h"
 
+//
+// API versions of this plugin engine
+//
+#define PLUGIN_API_VERSION_MAJOR 0
```

Comment:
Any reason for the move from plugin_info.h to plugin.h? The location matters less now that we have a good set off docs, but did it not work in the old location?

Reply:
I agree the previous location was better since usually the version changes when one changes `plugin_info.h`.

Reply:
Yup! I am trying to recall why i moved them, really :) 
I will fix it asap!

Reply:
FIxed. Thanks!